data/init.sql

```diff
@@ -0,0 +1,202 @@
+/*
+ * Set up the Initial Database for Us
+ * CON-In-A-Box 2018 Thomas Keeley
+ */
+
+-- SHOW DATABASES;
+-- SHOW TABLES;
+
+CREATE TABLE 'ConComList' (
+  'ListRecordID' INT UNSIGNED NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  'AccountID' INT UNSIGNED NOT NULL,
+      -- Taken from NeonCRM Currently
+  'Department' INT UNSIGNED NOT NULL,
+  'Position' INT UNSIGNED NOT NULL,
+  'Note' VARCHAR(100),
+  'ConventionYear' INT UNSIGNED NOT NULL
+);
+
+CREATE TABLE 'ConComPositions' (
+  'PositionID' INT UNSIGNED NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  'Name' VARCHAR(50) NOT NULL
+);
+
+INSERT INTO 'ConComPositions' ('Name') VALUES ('Head');
+INSERT INTO 'ConComPositions' ('Name') VALUES ('Sub-Head');
+INSERT INTO 'ConComPositions' ('Name') VALUES ('Specialist');
+
+CREATE TABLE 'ConventionYear' (
+  'YearID' INT UNSIGNED NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  'Name' VARCHAR(50) NOT NULL
+);
+
+INSERT INTO 'ConventionYear' ('Name') VALUES ('1999');
+INSERT INTO 'ConventionYear' ('Name') VALUES ('2000');
+INSERT INTO 'ConventionYear' ('Name') VALUES ('2001');
+INSERT INTO 'ConventionYear' ('Name') VALUES ('2002');
+INSERT INTO 'ConventionYear' ('Name') VALUES ('2003');
+INSERT INTO 'ConventionYear' ('Name') VALUES ('2004');
+INSERT INTO 'ConventionYear' ('Name') VALUES ('2005');
+INSERT INTO 'ConventionYear' ('Name') VALUES ('2006');
+INSERT INTO 'ConventionYear' ('Name') VALUES ('2007');
+INSERT INTO 'ConventionYear' ('Name') VALUES ('2008');
+INSERT INTO 'ConventionYear' ('Name') VALUES ('2009');
+INSERT INTO 'ConventionYear' ('Name') VALUES ('2010');
+INSERT INTO 'ConventionYear' ('Name') VALUES ('2011');
+INSERT INTO 'ConventionYear' ('Name') VALUES ('2012');
+INSERT INTO 'ConventionYear' ('Name') VALUES ('2013');
+INSERT INTO 'ConventionYear' ('Name') VALUES ('2014');
+INSERT INTO 'ConventionYear' ('Name') VALUES ('2015');
+INSERT INTO 'ConventionYear' ('Name') VALUES ('2016');
+INSERT INTO 'ConventionYear' ('Name') VALUES ('2017');
+INSERT INTO 'ConventionYear' ('Name') VALUES ('2018');
+
```

Comment:
More for the "fill con data" step and not the structure step.

I will not point them all out I think they are pretty clear.